### PR TITLE
Create 2018-06-11--js-ipfs-dev-team-weekly.md

### PR DESCRIPTION
**Pull request by fsdiogo**
Once the youtube recording link is provided I'll update it in the md.

### CODE REVIEW

**Comment by Mr0grog** [comment]
Anybody know what the situation is with the video here? It doesn’t seem like anything ever got uploaded (looking at https://www.youtube.com/channel/UCdjsUXJ3QawK4O5L1kqqsew/videos) — was there actually a recording?

**Comment by fsdiogo** [comment]
@jacobheun took the lead on this call, but apparently he just didn't have permissions to upload the video to youtube.

**Comment by jacobheun** [comment]
@Mr0grog yes I did a cloud recording of the meeting on Zoom, but it doesn't look like I have access to the recording as an alternate host. I believe I'll need the meeting owner, I believe that's only @diasdavid ?, to get the video in order to move that to youtube (as I also don't have access to it). 

Hopefully we can get live streaming of the meetings setup to make this easier going forward and get everyone access to the video sooner.

Sorry for the delay with this.

File: meeting-notes/2018/2018-06-11--js-ipfs-dev-team-weekly.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    - https://github.com/ipfs/js-ipfs/pull/1386
+    - Working on daemon stability
+- Blocked:
+    - N/A
+- Next:
+    - Daemon stability: add tests to reproduce errors
+
+@hugomrdias
+ - Done:
+    - make js-ipfs support `create-react-app`
```
Comment (diasdavid) [inline meeting-notes/2018/2018-06-11--js-ipfs-dev-team-weekly.md]: @hugomrdias is this finished? Is there an example/demo?

File: meeting-notes/2018/2018-06-11--js-ipfs-dev-team-weekly.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+ - Next:
+    - Continue the research on big files okr
+    - FileSystem API, idbMutableFile, Permanent storage permissions
+    - Streaming user download
+
+@gar
+ - Done:
+  - ipfsd-ctl bugfix https://github.com/ipfs/js-ipfsd-ctl/pull/247
+ - Blocked:
+ 	- ipfs-bitswap release for ledgerForPeer
```
Comment (diasdavid) [inline meeting-notes/2018/2018-06-11--js-ipfs-dev-team-weekly.md]: @vmx can you unblock @wraithgar ?

File: meeting-notes/2018/2018-06-11--js-ipfs-dev-team-weekly.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+## Notes
+
+### Week Update
+
+@jacobheun
+ - Done:
+ 	- Rolled back https://github.com/libp2p/js-libp2p-webrtc-star to 0.15.1 to resolve an issue with the block-stream update breaking js-ipfs and libp2p tests. Libp2p still needs to be released.
+  - Core functionality for private networks is in place, need to work on interface and interop tests for it.
+ - Blocked:
+ 	- Libp2p release is blocking https://github.com/libp2p/js-libp2p/issues/198 and https://github.com/ipfs/js-ipfs/issues/1379
```
Comment (diasdavid) [inline meeting-notes/2018/2018-06-11--js-ipfs-dev-team-weekly.md]: Added to my Tuesday TODO

File: meeting-notes/2018/2018-06-11--js-ipfs-dev-team-weekly.md (hunk 1 above)
Comment (hugomrdias) [reply meeting-notes/2018/2018-06-11--js-ipfs-dev-team-weekly.md]: i think we still need a libp2p and ipfs release.

File: meeting-notes/2018/2018-06-11--js-ipfs-dev-team-weekly.md (hunk 2 above)
Comment (vmx) [reply meeting-notes/2018/2018-06-11--js-ipfs-dev-team-weekly.md]: I can't do a release, I'm not an owner of the ipfs-bitswap module.

Comment (vmx) [reply meeting-notes/2018/2018-06-11--js-ipfs-dev-team-weekly.md]: @wraithgar Done: https://github.com/ipfs/js-ipfs-bitswap/commit/72fc29d30c37e3bc85ad079628bd769bb1d3df63

File: meeting-notes/2018/2018-06-11--js-ipfs-dev-team-weekly.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+- **Lead:** Jacob Heun (@jacobheun)
+- **Notetaker:** Diogo Silva (@fsdiogo)
+- **Attendees:**
+  - @achingbrain
+  - @fsdiogo
+  - @vasco-santos
+  - @hugomrdias
+  - @gar
+- **Recording:** _add link to recording once it's online_
```
Comment (diasdavid) [inline meeting-notes/2018/2018-06-11--js-ipfs-dev-team-weekly.md]: @jacobheun @fsdiogo where is the recording for this call?

Comment (fsdiogo) [reply meeting-notes/2018/2018-06-11--js-ipfs-dev-team-weekly.md]: Check this https://github.com/ipfs/pm/pull/643#issuecomment-398060247.

Comment (jacobheun) [reply meeting-notes/2018/2018-06-11--js-ipfs-dev-team-weekly.md]: @diasdavid the recording should be in your cloud recordings, https://protocol.zoom.us/recording. Sorry about that, I selected cloud recording and incorrectly assumed I would have access to it, but it only looks like alternate hosts can start/stop the recording. I will use the local recording in the future.

**Comment by diasdavid** [comment]
Added the recording to the notes. Url here as well https://youtu.be/N6-rxdV3K9s

**Pull request merged**